src/test/resources/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14SwitchExpression.txt

```diff
@@ -1269,6 +1269,7 @@ CLASS_DEF -> CLASS_DEF [13:0]
     |       |   |           |       |   |       |--LITERAL_INSTANCEOF -> instanceof [168:25]
     |       |   |           |       |   |       |   |--IDENT -> o1 [168:22]
     |       |   |           |       |   |       |   `--PATTERN_VARIABLE_DEF -> PATTERN_VARIABLE_DEF [168:36]
```

Comment:
Isn't it wrong that this is in a java 14 file?

Reply:
Syntactically, yes, but having an empty/ dummy `MODIFIERS` ast here isn't really hurting anything. We use the `MODIFIERS` ast in this way for all constructs (it is always created, even if no modifiers exist). Pattern variables were still in preview until Java 16, so I don't think that it is a big deal that we just stick with final implementation.